src/lsp/server/diagnostics/html/mobileOfflineFriendly.ts

```diff
@@ -17,11 +17,19 @@ const baseComponentsAttributes = { values: getBaseComponentsAttributes() };
 
 export const RULE_ID = 'mobile-offline-friendly';
 
-export class MobileOfflineFriendly implements DiagnosticProducer<HTMLDocument> {
+export class MobileOfflineFriendly extends DiagnosticProducer<HTMLDocument> {
+    constructor(baseDocUrl: string) {
+        super(baseDocUrl);
+    }
+
     getId(): string {
         return RULE_ID;
     }
 
+    hasDocumentation(): boolean {
+        return false;
+    }
```

Comment:
note: MobileOfflineFriendly diagnostic does not has its own MD file. The tool tip for component from other SF tool has the offline indicator within the doc.